test/runtime-utils/src/lib.rs

```diff
@@ -0,0 +1,129 @@
+#![feature(step_trait)]
```

Comment:
Why do we need this nightly feature?

Reply:
the type in the for loop used to be generic, I changed that and forgot to remove the feature guard.